python/GafferSceneUI/LightEditor.py

```diff
@@ -146,7 +146,8 @@ def registerParameter( cls, rendererKey, parameter, section = None, columnName =
 		# We use `tuple` to store `ShaderNetwork.Parameter`, because
 		# the latter isn't hashable and we need to use it as a dict key.
 		if isinstance( parameter, str ) :
-			parameter = ( "", parameter )
+			shader, dot, param = parameter.rpartition( "." )
```

Comment:
I initially went with `rpartition()` because `"parameter".partition(".") = ( "parameter", "", "" )`, so it would look like the shader name is `parameter`. I think the most correct solution is `partition()` but only if there's a `.` found in the string?